src/main/java/org/jugistanbul/secondopinion/api/utils/ObjectUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.jugistanbul.secondopinion.api.utils;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.BeanWrapper;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class ObjectUtils {
+
+  // then use Spring BeanUtils to copy and ignore null
+  public static void myCopyProperties(Object src, Object target) {
```

Comment:
yordam `myCopyProperties` adı yordamın ne işe yaradığını ortaya koymuyor gibi; bu yüzden de açıklama satırı ile *null değerlerin gözardı edildiği* bilgisi verilmek durumunda kalınmış. Açıklama satırını okumak için de yordamın bulunduğu kodu açmak gerekiyor. Yordam için `copyNonNullProperties` adı daha uygun olacaktır.

Reply:
tamamdir. bu sekilde adlandiralim.